tests/rptest/tests/e2e_topic_recovery_test.py

```diff
@@ -216,6 +219,7 @@ def done():
                     producer.produce(self.topic)
                 except ck.KafkaException as err:
                     self.logger.warn(f"producer error: {err}")
+                    time.sleep(10)
```

Comment:
> Use short timeout if kafka producer returned an error. Add exception for
BadLogLines.

This commit message is describing the literal introduction of the sleep, not why the sleep is being added. Can you add the _why_ to the commit message?

Reply:
thanks!